web/src/config/client_swagger.json

```diff
@@ -1693,100 +1693,7 @@
-          },
-          "401": {
-            "description": "Unauthorized: Token has not found or invalid format or session is invalid"
-          },
-          "499": {
-            "description": "Call Cancelled: Call cancelled by client."
-          },
-          "500": {
-            "description": "Internal Server Error: When requested resource is not available."
-          }
-        },
-        "content": {
-          "application/json": {}
-          }
-      }
-    },
-    "/system/health/node": {
-      "get": {
-        "tags": ["Health"],
-        "summary": "Get Health of nodes: Accessible to CSM users having monitor and manage permissions",
-        "responses": {
-          "200": {
-            "description": "OK: Health of the nodes fetched successfully."
-          },
-          "400": {
-            "description": "Bad Request: when an invalid response message is received for any of the cli commands."
-          },
-          "401": {
-            "description": "Unauthorized: Token has not found or invalid format or session is invalid"
-          },
-          "403": {
-            "description": "Forbidden: This error is raised for all cases when user don't have permissions"
-          },
-          "499": {
-            "description": "Call Cancelled: Call cancelled by client."
-          },
-          "500": {
-            "description": "Internal Server Error: When requested resource is not available."
-          }
-        },
-        "content": {
-          "application/json": {}
-          }
-      }
-    },
-    "/system/health/components": {
-      "get": {
-        "tags": ["Health"],
-        "summary": "Get components of nodes and their health: Accessible to CSM users having monitor and manage permissions",
-        "responses": {
-          "200": {
-            "description": "OK: Health of the nodes fetched successfully."
-          },
-          "400": {
-            "description": "Bad Request: when an invalid response message is received for any of the cli commands."
-          },
-          "401": {
-            "description": "Unauthorized: Token has not found or invalid format or session is invalid"
-          },
-          "403": {
-            "description": "Forbidden: This error is raised for all cases when user don't have permissions"
-          },
-          "499": {
-            "description": "Call Cancelled: Call cancelled by client."
-          },
-          "500": {
-            "description": "Internal Server Error: When requested resource is not available."
-          }
-        },
-        "content": {
-          "application/json": {}
-        },
-        "parameters": [
-          {
-            "name": "node_id",
-            "in": "query",
-            "description": "Specifies the node id",
-            "required": false,
-            "schema": { "type": "string" }
-          }
-        ]
-      }
-    },
-    "/system/health/resources": {
+    "/system/health/{resource}": {
```

Comment:
does its entry exist in csm-web too as a passthrough layer... if yes.. we need to remove that. Please confirm

Reply:
Yes Pranay, there were some routes and controllers related to this. I have removed them now.